website/client/src/components/HeaderBar/UserAva.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Button from '../Button/Button';
+
+export default class UserAva extends React.Component {
+
+    handleAccountSetting = () => {
+
+    }
+
+    handleLogout = () => {
+        localStorage.clear();
+        window.location.reload();
```

Comment:
window.location.reload is ok, but there is an official prop from react-router name "history", you can use history.push("/"). Or if you want to remove the "user" from redux, just call an action.
Ref: https://reacttraining.com/react-router/web/api/history